index.php

```diff
@@ -1 +1,48 @@
 
+<?php
+
+/*
+ * Complete the cookies function below.
+ */
+function cookies($k, $A) {
+$step = 0;
+do{
+    $step++;
+    if(count($A) < 2) return -1;
```

Comment:
Please use code formatting (autoformatting in PHPStorm, you can even setup PSR-12 style there)
Also please check https://www.php-fig.org/psr/psr-12/#5-control-structures
`The body of each structure MUST be enclosed by braces`